### PR TITLE
Document sqlTeamStore get methods

### PR DESCRIPTION
**Pull request by FlaviaBastos**
#### Summary
Documenting 'sqlTeamStore' Get methods.

#### Ticket Link
 Fixes https://github.com/mattermost/mattermost-server/issues/14007

JIRA: [https://mattermost.atlassian.net/browse/MM-22999](https://mattermost.atlassian.net/browse/MM-22999)

### CODE REVIEW

File: store/sqlstore/team_store.go [hunk 1]
```diff
@@ -304,6 +307,9 @@ func (s SqlTeamStore) Get(id string) (*model.Team, *model.AppError) {
 	return obj.(*model.Team), nil
 }
 
+// GetByInviteId returns from the database the team that matches the invitedId provided as parameter.
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:310]:
```suggestion
// GetByInviteId returns from the database the team that matches the inviteId provided as parameter.
```

Comment (FlaviaBastos) [reply store/sqlstore/team_store.go]: Just pushed change up.

**Comment by teresa-novoa** [comment]
@jespino can you help to merge this PR? Thanks!

**Comment by teresa-novoa** [comment]
Thanks @jespino!

**Pull request merged**